keycloak/themes/hmda/login/register.ftl

```diff
@@ -39,12 +39,36 @@
             <div class="usa-alert-text">
               <p>Passwords must:</p>
               <ul id="validation_list">
-                <li data-validator="length">Be at least 12 characters</li>
-                <li data-validator="uppercase">Have at least 1 uppercase character</li>
-                <li data-validator="lowercase">Have at least 1 lowercase character</li>
-                <li data-validator="numerical">Have at least 1 numerical character</li>
-                <li data-validator="special">Have at least 1 special character</li>
-                <li data-validator="username">Not be the same as your username</li>
+                <li data-validator="length">
+                  <img class="check" src="${url.resourcesPath}/img/correct9.png">
+                  <img class="missing" src="${url.resourcesPath}/img/close.png">
+                  Be at least 12 characters
+                </li>
+                <li data-validator="uppercase">
+                  <img class="check" src="${url.resourcesPath}/img/correct9.png">
+                  <img class="missing" src="${url.resourcesPath}/img/close.png">
+                  Have at least 1 uppercase character
+                </li>
+                <li data-validator="lowercase">
```

Comment:
Is it possible to use `background` on the `li` to update the icons? Then we could remove the `<img>` tags from here.

So, `<li class="missing">` could get a style of `background: url('theX.png') no-repeat ...`.

Reply:
Ahh, I see. I think this ok. 👍 

Maybe we can investigate it later.